CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.0.2 (2019-05-17)
+
+- Fixed autocomplete component to emit a `selectionChange` event when input is cleared. [#22](https://github.com/blackbaud/skyux-lookup/issues/22)
```

Comment:
We're back on with articles, yeah? 

Fixed the autocomplete component to emit a `selectionChange` event when the input is cleared.